cypress/integration/v2/features/global-settings/global-typography2.spec.js

```diff
@@ -0,0 +1,351 @@
+/**
+ * External dependencies
+ */
+import { range } from 'lodash'
+import { registerBlockSnapshots } from '~gutenberg-e2e/plugins'
+import {
+	 responsiveAssertHelper,
+	 registerTests,
+} from '~stackable-e2e/helpers'
+
+import { blocks } from '~stackable-e2e/config'
+
+const [ desktopGlobal, tabletGlobal, mobileGlobal ] = responsiveAssertHelper( globalTypography, { tab: 'Global Typography' } )
+
+describe( 'Global Settings', registerTests( [
+	 desktopGlobal,
+	 tabletGlobal,
+	 mobileGlobal,
+] ) ) //Viewports
+
+function globalTypography( viewport, desktopOnly ) {
+	 cy.setupWP()
+	 cy.newPage()
+
+	 const globalTypo = [
+		 {
+			 tag: 'h1',
+			 font: 'Abel',
+			 size: 92,
+			 weight: 'bold',
+			 transform: 'uppercase',
+			 lineHeight: 1.3,
+			 letterSpacing: 2.1,
+		 },
+		 {
+			 tag: 'h2',
+			 font: 'Aclonica',
+			 size: 66,
+			 weight: '900',
+			 transform: 'capitalize',
+			 lineHeight: 1.2,
+			 letterSpacing: 1.1,
+		 },
+		 {
+			 tag: 'h3',
+			 font: 'ABeeZee',
+			 size: 50,
+			 weight: '100',
+			 transform: 'lowercase',
+			 lineHeight: 1.1,
+			 letterSpacing: 1.5,
+		 },
+		 {
+			 tag: 'h4',
+			 font: 'Acme',
+			 size: 38,
+			 weight: '300',
+			 transform: 'uppercase',
+			 lineHeight: 2.7,
+			 letterSpacing: 1.8,
+		 },
+		 {
+			 tag: 'h5',
+			 font: 'Akronim',
+			 size: 24,
+			 weight: '500',
+			 transform: 'capitalize',
+			 lineHeight: 2.1,
+			 letterSpacing: 3.9,
+		 },
+		 {
+			 tag: 'h6',
+			 font: 'Actor',
+			 size: 16,
+			 weight: '600',
+			 transform: 'uppercase',
+			 lineHeight: 1.4,
+			 letterSpacing: 3.5,
+		 },
+		 {
+			 tag: 'p',
+			 font: 'Alice',
+			 size: 15,
+			 weight: '200',
+			 transform: 'lowercase',
+			 lineHeight: 1.3,
+			 letterSpacing: 0.1,
+		 },
+	 ]
+
+	 const blocksWithTitle = [
+		 'ugb/accordion',
+		 'ugb/heading',
+		 'ugb/text',
+		 'ugb/icon',
+		 'ugb/feature-grid',
+		 'ugb/image-box',
+		 'ugb/feature',
+		 'ugb/cta',
+		 'ugb/card',
+		 'ugb/header',
+		 'ugb/count-up',
+		 'ugb/pricing-box',
+		 'ugb/notification',
+		 'ugb/number-box',
+		 'ugb/expand',
+	 ]
+
+	 const blocksWithBlockTitle = [
+		 'ugb/columns',
+		 'ugb/icon-list',
+		 'ugb/video-popup',
+		 'ugb/testimonial',
+		 'ugb/team-member',
+	 ]
```

Comment:
we don't need to add this on Native blocks test because these are stackable blocks

Reply:
create an outline for the test first so it can be your guide when writing the code 😀
something like this -
**Global typography on native blocks:**
- adjust global typography using the `cy.adjustGlobalTypography` command
- for each native blocks (heading, paragraph, list), do the following:
1. add the native block in the editor
2. assert the typography styles added in Global typography settings for each native block